src/theme.ts

```diff
@@ -2,6 +2,14 @@ import { createMuiTheme } from "@material-ui/core/styles";
 
 const defaultTheme = createMuiTheme();
 
+// TODO: Toggle button theme
+// toggleButton: {
+//   "&.Mui-selected": {
+//     color: "#5A19B9",
+//     backgroundColor: "#CCA8FF",
+//   },
+// },
```

Comment:
let's remove this commented-out code! we can always find this PR again to look for the code if we need it